addons/sale_purchase/tests/test_sale_purchase.py

```diff
@@ -276,3 +276,62 @@ def test_pol_description(self):
 
         po = self.env['purchase.order'].search([('partner_id', '=', self.partner_vendor_service.id)], order='id desc', limit=1)
         self.assertEqual(po.order_line.name, "[C01] Name01")
+
+    def test_pol_custom_attribute(self):
+        """
+         test that custom atributes are passed from the SO the PO for service products
+        """
+        # Setup service product variants
+        service = self.env['product.product'].create({
+            'name': 'Super Product',
+            'type': 'service',
+            'service_to_purchase': True,
+            'seller_ids': [(0, 0, {
+                'name': self.partner_vendor_service.id,
+                'min_qty': 1,
+                'price': 10,
+                'sequence': 1,
+            })]
+        })
+
+        product_attribute = self.env['product.attribute'].create({
+            'name': 'product attribute',
+            'display_type': 'radio',
+            'create_variant': 'always'
+        })
+
+        product_attribute_value = self.env['product.attribute.value'].create({
+            'name': 'single product attribute value',
+            'is_custom': True,
+            'attribute_id': product_attribute.id
+        })
+
+        product_attribute_line = self.env['product.template.attribute.line'].create({
+            'attribute_id': product_attribute.id,
+            'product_tmpl_id': service.product_tmpl_id.id,
+            'value_ids': [(6, 0, [product_attribute_value.id])]
```

Comment:
```suggestion
            'value_ids': [Command.link(product_attribute_value.id)]
```
No need to do a `SET` here, as no `value_ids` need to be unlinked.